packages/logging/system-test/logging.js

```diff
@@ -351,8 +351,8 @@ describe('Logging', function() {
       var entry1 = log.entry('1');
 
       setTimeout(function() {
-        var entry3 = log.entry('3');
-        var entry2 = log.entry({ timestamp: entry3.metadata.timestamp }, '2');
+        var entry2 = log.entry('2');
+        var entry3 = log.entry({ timestamp: entry2.metadata.timestamp }, '3');
```

Comment:
I should have caught this earlier too – I had been running `mocha` with `--grep` with my own test...